developer/src/kmc-ldml/test/fixtures/basic.js

```diff
@@ -5,7 +5,7 @@ if(typeof keyman === 'undefined') {
 }
 function Keyboard_basic() {
   this.KI="Keyboard_basic";
-  this.KN={"value":"TestKbd"};
+  this.KN="TestKbd";
```

Comment:
Yeah ... this is more correct. It should never have been an object!

Reply:
`JSON.stringify()` was used to good effect to escape for .js code.  but it was a `StrsItem` object